webclient/package.json

```diff
@@ -30,8 +31,8 @@
     "typescript": "^4.4.4"
   },
   "scripts": {
-    "postinstall": "run-script-os",
-    "postinstall:windows": "powershell .\\copy_shared_files.ps1",
+    "postinstall": "run-script-os && npm run prepare",
+    "postinstall:windows": "powershell ./copy_shared_files.ps1",
```

Comment:
I was having issues running postinstall with the forward slash. On windows powershell, it would fail with the double forward slash and pass with the change